packages/ui/src/composables/useSize.ts

```diff
@@ -0,0 +1,117 @@
+import { computed, PropType } from 'vue'
+import { getGlobalConfig, SizeConfig } from '../services/global-config/global-config'
+
+export const sizesConfig: SizeConfig = {
+  defaultSize: 48,
+  sizes: {
+    small: 32,
+    medium: 48,
+    large: 64,
+  },
+}
+
+export const fontSizesConfig: SizeConfig = {
+  defaultSize: 1,
+  sizes: {
+    small: 0.75,
+    medium: 1,
+    large: 1.25,
+  },
+}
+
+interface SizeProps {
+  size: string | number;
+  sizesConfig: Record<string, any>;
+  fontSizesConfig: Record<string, any>;
+}
+
+/**
+ * You could add these props to any component by destructuring them inside props option.
+ * @example
+ * props: { ...useSizeProps, componentsOwnProp, etc. }
+ * It's better to add props at the beginning, to make sure that Component own props will be used instead in case of collision
+ */
+export const useSizeProps = {
+  size: {
+    type: [String, Number] as PropType<string | number>,
+    default: '',
+    validator: (size: string | number) => {
+      return typeof size === 'string' || typeof size === 'number'
+    },
+  },
+
+  sizesConfig: {
+    type: Object as PropType<Record<string, any>>,
+    default: () => sizesConfig,
+  },
+
+  fontSizesConfig: {
+    type: Object as PropType<Record<string, any>>,
+    default: () => fontSizesConfig,
+  },
+}
+
+const fontRegex = /(?<fontSize>\d+)(?<extension>px|rem)/i
+
+const convertToRem = (px: number) => px / 16 - 0.5
+
+export const useSize = (
+  props: SizeProps,
+  componentName?: string | undefined,
+) => {
+  const sizesConfigGlobal = computed<SizeConfig>(() => {
+    return componentName
+      ? getGlobalConfig().components?.[componentName]?.sizesConfig
+      : undefined
+  })
+
+  const sizeComputed = computed<string>(() => {
+    const { defaultSize, sizes } = props.sizesConfig
+    const defaultSizeGlobal = sizesConfigGlobal.value?.defaultSize
+
+    if (!props.size) {
+      return defaultSizeGlobal
```

Comment:
``return `${defaultSizeGlobal || defaultSize}px` ``

Reply:
Fixed it